internal/models/sessions.go

```diff
@@ -118,6 +118,14 @@ func FindSessionsByFactorID(tx *storage.Connection, factorID uuid.UUID) ([]*Sess
 	return sessions, nil
 }
 
+func FindSessionsByUserID(tx *storage.Connection, userID uuid.UUID) ([]*Session, error) {
+	sessions := []*Session{}
+	if err := tx.Q().Where("user_id = ?", userID).All(&sessions); err != nil {
+		return nil, err
+	}
+	return sessions, nil
+}
+
```

Comment:
hmm this doesn't seemed to be used anywhere except in the tests?

Reply:
Yes, it is only used in tests - we can move to `*_test.go` as a test helper if that'd be better. Lmk